extensions/gitpod/src/remoteConnector.ts

```diff
@@ -629,9 +629,12 @@ export default class RemoteConnector extends Disposable {
 						this.logger.error(`Failed to connect to ${params.workspaceId} Gitpod workspace`, e);
 					}
 					const seeLogs = 'See Logs';
-					const action = await vscode.window.showErrorMessage(`Failed to connect to ${params.workspaceId} Gitpod workspace`, seeLogs);
+					const showTroubleshooting = 'Show Troubleshooting';
+					const action = await vscode.window.showErrorMessage(`Failed to connect to ${params.workspaceId} Gitpod workspace`, seeLogs, showTroubleshooting);
 					if (action === seeLogs) {
 						this.logger.show();
+					} else if (action === showTroubleshooting) {
+						vscode.env.openExternal(vscode.Uri.parse('https://www.gitpod.io/docs/troubleshooting'));
```

Comment:
we need to point to SSH docs which should explain and then lead to troubleshooting

Reply:
Then we need to wait for the SSH docs PR to be merged first